templates/install/windows.md

```diff
@@ -62,10 +62,15 @@ In order to use mathlib supporting tools, you need to [get python](https://www.p
 
 ### Get Scripts
 
-Then, at a terminal, run the command
-  ```bash
-  pip3 install mathlibtools
-  ```
+Then we install a small tool called `leanproject` that which will handle
+updating mathlib according to the needs of your current project. We use
+[pipx](https://pipxproject.github.io/pipx/) to perform the installation.
+At a terminal, run the commands
+```bash
+python3 -m pip install --user pipx
+python3 -m pipx ensurepath
+pipx install mathlibtools
```

Comment:
Sadly, this doesn't work. I just followed the instructions on a fresh Windows 10 machine, and upon getting to this step `pipx install mathlibtools` just says `bash: pipx: command not found`.

I closed and restarted "git bash", and get the same message. I rebooted the windows machine (I have not yet tried unplugging the router...) and get the same message.

Reply:
Thank you for testing this. I'm looking into it.

Reply:
The first problem is that `pipx ensurepath` modifies the Windows `PATH` variable, but we want to modify the `git bash` `PATH`, which are different.

After manually setting the path, there was also some `venv` problem, which can probably be fixed, but I didn't try very hard.